server/controller/routes/user/getUser.js

```diff
@@ -0,0 +1,19 @@
+const boomify = require('../../../utils/boomify');
+const { getUserData } = require('../../../database/queries/user');
+
+const getUser = async (req, res, next) => {
+  try {
+    const { userId } = req;
+    const { rows } = await getUserData(userId);
+
+    if (rows.length === 0) {
+      throw boomify(404, 'User Not Found!');
```

Comment:
on this issue, if the query returns [ ], the problem on the database so its server error  500, 'database error'